### DISCUSSION TITLE
Issue on PostInstallScript

### DISCUSSION THREAD

**Discussion by srikanth-6520** [Sunbird-Ed/Community#516] [Bugs] [answered]
Hi Team,
 we are getting an error while running the postinstalltionScript in jenkins , we have provided jenkins console below,
 
Started by user [Kolluri Srikanth]
Rebuilds build #[3](https://jenkins.alex-sb.dev.verlab.io/job/OpsAdministration/job/alex/job/dev/job/Core/job/PostInstallScript/3)
Checking out git https://github.com/project-sunbird/sunbird-devops.git into /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript@script/74766eba9651ee97692d587da848d14cae2c225f81df644eb5a529f729d5e85a to read kubernetes/pipelines/post-install/Jenkinsfile
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript@script/74766eba9651ee97692d587da848d14cae2c225f81df644eb5a529f729d5e85a/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-devops.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from https://github.com/project-sunbird/sunbird-devops.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress --depth=1 -- https://github.com/project-sunbird/sunbird-devops.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.1.0^{commit} # timeout=10
Checking out Revision bee2ee6f39baa087eba22646d7919493552b10e1 (origin/release-5.1.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bee2ee6f39baa087eba22646d7919493552b10e1 # timeout=10
Commit message: "Merge pull request #3793 from project-sunbird/gandham-santhosh-patch-1"
First time build. Skipping changelog.
Loading library deploy-conf@heads/shared-lib
Attempting to resolve heads/shared-lib from remote references...
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git ls-remote -- https://github.com/project-sunbird/sunbird-devops.git # timeout=10
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript@libs/b4ac4d1235c67cfedda9d7bf40936b35bc6929460ef19107abe1b183748d4145/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-devops.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from https://github.com/project-sunbird/sunbird-devops.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --no-tags --progress --depth=1 -- https://github.com/project-sunbird/sunbird-devops.git +refs/heads/*:refs/remotes/origin/* refs/heads/shared-lib # timeout=10
Checking out Revision 778da7ce7f97efd003eed98071921a837d209343 (778da7ce7f97efd003eed98071921a837d209343)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 778da7ce7f97efd003eed98071921a837d209343 # timeout=10
Commit message: "fix: move variables to be accessible across both try blocks"
 > git rev-list --no-walk 778da7ce7f97efd003eed98071921a837d209343 # timeout=10
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [Jenkins](https://jenkins.alex-sb.dev.verlab.io/computer/(built-in)/) in /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript
[Pipeline] {
[Pipeline] stage
[Pipeline] { (checkout public repo)
[Pipeline] echo
Found .git folder. Clearing it..
[Pipeline] sh
+ git clean -fxd
Removing ansible/inventory/env/hosts
Removing ansible/inventory/env/keys/
Removing ansible/inventory/env/secrets.yml
Skipping repository private/
Removing private@tmp/
[Pipeline] checkout
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-devops.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from https://github.com/project-sunbird/sunbird-devops.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
 > git fetch --tags --progress --depth=1 -- https://github.com/project-sunbird/sunbird-devops.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.1.0^{commit} # timeout=10
Checking out Revision bee2ee6f39baa087eba22646d7919493552b10e1 (origin/release-5.1.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bee2ee6f39baa087eba22646d7919493552b10e1 # timeout=10
Commit message: "Merge pull request #3793 from project-sunbird/gandham-santhosh-patch-1"
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (deploy)
[Pipeline] sh
+ echo OpsAdministration/alex/dev/Core/PostInstallScript
[Pipeline] sh
+ echo OpsAdministration/alex/dev/Core/PostInstallScript
[Pipeline] sh
+ echo OpsAdministration/alex/dev/Core/PostInstallScript
[Pipeline] sh
+ pwd
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (ansible-run)
[Pipeline] echo
{currentWs=/var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript, env=dev, module=Core, ansiblePlaybook=/var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/ansible/post-install.yaml, ansibleExtraArgs=--vault-password-file /var/lib/jenkins/secrets/vault-pass -v}
[Pipeline] dir
Running in /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/private
[Pipeline] {
[Pipeline] echo
Info: Branch override is enabled
[Pipeline] checkout
The recommended git tool is: NONE
using credential bitbucket-cred
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/private/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url bitbucket.org:verlab/veranda-alex-sb-private.git # timeout=10
Using shallow fetch with depth 1
Fetching upstream changes from bitbucket.org:verlab/veranda-alex-sb-private.git
 > git --version # timeout=10
 > git --version # 'git version 2.17.1'
using GIT_SSH to set credentials Veranda Bitbucket Credential (https://bitbucket.org/verlab/veranda-alex-sb-private)
Verifying host key using known hosts file
 > git fetch --no-tags --progress --depth=1 -- bitbucket.org:verlab/veranda-alex-sb-private.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/release-5.1.0^{commit} # timeout=10
Checking out Revision 5f8ed7daa1c962fd3ee3a498ccc2a42fc6eae1fd (origin/release-5.1.0)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 5f8ed7daa1c962fd3ee3a498ccc2a42fc6eae1fd # timeout=10
Commit message: "we have added opa variabes "
 > git rev-list --no-walk 09cb611d7202632f7a2864b514017814ce5af233 # timeout=10
[Pipeline] }
[Pipeline] // dir
[Pipeline] sh
+ rsync -Lkr /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/private/ansible/inventory/dev/Core/common.yml /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/private/ansible/inventory/dev/Core/hosts /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/private/ansible/inventory/dev/Core/keys /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/private/ansible/inventory/dev/Core/secrets.yml /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/ansible/inventory/env/
+ [ -f /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/ansible/inventory/env/kubernetes.yaml ]
+ ansible-playbook -i /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/ansible/inventory/env/hosts /var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/ansible/post-install.yaml --vault-password-file /var/lib/jenkins/secrets/vault-pass -v
No config file found; using defaults
[WARNING]: Invalid characters were found in group names but not replaced, use
-vvvv to see details

[WARNING]:  * Failed to parse /var/lib/jenkins/workspace/OpsAdministration/alex
/dev/Core/PostInstallScript/ansible/inventory/env/hosts with yaml plugin:
Syntax Error while loading YAML.   found unexpected ':'  The error appears to
be in '/var/lib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallSc
ript/ansible/inventory/env/hosts': line 1, column 5, but may be elsewhere in
the file depending on the exact syntax problem.  The offending line appears to
be:   [all:vars]     ^ here

[WARNING]:  * Failed to parse /var/lib/jenkins/workspace/OpsAdministration/alex
/dev/Core/PostInstallScript/ansible/inventory/env/hosts with ini plugin: /var/l
ib/jenkins/workspace/OpsAdministration/alex/dev/Core/PostInstallScript/ansible/
inventory/env/hosts:150: Section [core:children] includes undefined group:
postgresql-slave

[WARNING]: Unable to parse /var/lib/jenkins/workspace/OpsAdministration/alex/de
v/Core/PostInstallScript/ansible/inventory/env/hosts as an inventory source

[WARNING]: No inventory was parsed, only implicit localhost is available

[WARNING]: provided hosts list is empty, only localhost is available. Note that
the implicit localhost does not match 'all'

PLAY [localhost] ***************************************************************

TASK [post-install : uri] ******************************************************
[DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host localhost should use 
/usr/bin/python3, but is using /usr/bin/python for backward compatibility with 
prior Ansible releases. A future Ansible release will default to using the 
discovered platform python for this host. See [https://docs.ansible.com/ansible/](https://docs.ansible.com/ansible/%1B[0m)
2.8/reference_appendices/interpreter_discovery.html for more information. This 
feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
ok: [localhost] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "cache_control": "no-store",
    "changed": false,
    "connection": "close",
    "content_length": "2676",
    "content_type": "application/json",
    "cookies": {},
    "cookies_string": "",
    "date": "Wed, 07 Jun 2023 05:07:56 GMT",
    "elapsed": 0,
    "json": {
        "access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI4NklnYTA1UUVXRWxOcF9aMzkyQ3BzZDBTX3gzWm1nZkczNFp3cGFza2pRIn0.eyJqdGkiOiJiNmQ0NzI3OS05MDMzLTRkNTAtYWY0YS0yMjlmZDUxZjNjYmMiLCJleHAiOjE2ODYxNTc2NzYsIm5iZiI6MCwiaWF0IjoxNjg2MTE0NDc2LCJpc3MiOiJodHRwczovL2FsZXgtc2IuZGV2LnZlcmxhYi5pby9hdXRoL3JlYWxtcy9zdW5iaXJkIiwiYXVkIjpbInJlYWxtLW1hbmFnZW1lbnQiLCJhY2NvdW50Il0sInN1YiI6IjI5NTAwYmY3LTg2NGYtNDFhYS05MGE4LWM0MjBlOWUwN2JkMCIsInR5cCI6IkJlYXJlciIsImF6cCI6ImxtcyIsImF1dGhfdGltZSI6MCwic2Vzc2lvbl9zdGF0ZSI6IjRjMzI4N2I0LWYyMWItNGNiNS1hNzZhLTBjMjU1MjkyZTBkYyIsImFjciI6IjEiLCJhbGxvd2VkLW9yaWdpbnMiOlsiaHR0cHM6Ly9hbGV4LXNiLmRldi52ZXJsYWIuaW8iXSwicmVhbG1fYWNjZXNzIjp7InJvbGVzIjpbIm9mZmxpbmVfYWNjZXNzIiwidW1hX2F1dGhvcml6YXRpb24iXX0sInJlc291cmNlX2FjY2VzcyI6eyJyZWFsbS1tYW5hZ2VtZW50Ijp7InJvbGVzIjpbIm1hbmFnZS11c2VycyJdfSwibG1zIjp7InJvbGVzIjpbInVtYV9wcm90ZWN0aW9uIl19LCJhY2NvdW50Ijp7InJvbGVzIjpbIm1hbmFnZS1hY2NvdW50IiwibWFuYWdlLWFjY291bnQtbGlua3MiLCJ2aWV3LXByb2ZpbGUiXX19LCJzY29wZSI6IiIsImNsaWVudElkIjoibG1zIiwiY2xpZW50SG9zdCI6IjE3Mi4xNi4yLjE3MiIsInByZWZlcnJlZF91c2VybmFtZSI6InNlcnZpY2UtYWNjb3VudC1sbXMiLCJjbGllbnRBZGRyZXNzIjoiMTcyLjE2LjIuMTcyIiwiZW1haWwiOiJzZXJ2aWNlLWFjY291bnQtbG1zQHBsYWNlaG9sZGVyLm9yZyJ9.UF-T3xE-pUKleE0LCBBcpJwjKO9MWE0nbjCi67JnezVvU4jPvXceKr2wcGySf-2tydaIG12zVwsKGRUCTD_DT1dDbeSriJztxhCBXELRu9AycJ489J6doqrFUQECFrESCQeqPkjQ8Jn1tjHzeqqFjZvn8dATrGrYUd2HZmP5auMFHv8N3cgrlgBkLwgHQYU_Qk5IgpcfViEqGMNZf1utZkPPeoh4RFckpD4q6IssN82XUv4VzbgiHW6dTgJvQfzhhY7kqlPZVvZApFYwalMS-1OLit5oJT5ynTynIXf4oJoTNpiCBVAf9TVnPEl9Xy-pQHnsNJ1IAKWff-Y4fIHacQ",
        "expires_in": 43200,
        "not-before-policy": 1548955037,
        "refresh_expires_in": 86400,
        "refresh_token": "eyJhbGciOiJIUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI3NDc2ZDVkNC1lMzE4LTQ4ZjctYjdjNi0xNjJiZDQ4MDE1NDkifQ.eyJqdGkiOiI1NGJjZTE3Mi02NzlkLTQzOWYtYmViOC1jMWQ0NThlN2NiOTUiLCJleHAiOjE2ODYyMDA4NzYsIm5iZiI6MCwiaWF0IjoxNjg2MTE0NDc2LCJpc3MiOiJodHRwczovL2FsZXgtc2IuZGV2LnZlcmxhYi5pby9hdXRoL3JlYWxtcy9zdW5iaXJkIiwiYXVkIjoiaHR0cHM6Ly9hbGV4LXNiLmRldi52ZXJsYWIuaW8vYXV0aC9yZWFsbXMvc3VuYmlyZCIsInN1YiI6IjI5NTAwYmY3LTg2NGYtNDFhYS05MGE4LWM0MjBlOWUwN2JkMCIsInR5cCI6IlJlZnJlc2giLCJhenAiOiJsbXMiLCJhdXRoX3RpbWUiOjAsInNlc3Npb25fc3RhdGUiOiI0YzMyODdiNC1mMjFiLTRjYjUtYTc2YS0wYzI1NTI5MmUwZGMiLCJyZWFsbV9hY2Nlc3MiOnsicm9sZXMiOlsib2ZmbGluZV9hY2Nlc3MiLCJ1bWFfYXV0aG9yaXphdGlvbiJdfSwicmVzb3VyY2VfYWNjZXNzIjp7InJlYWxtLW1hbmFnZW1lbnQiOnsicm9sZXMiOlsibWFuYWdlLXVzZXJzIl19LCJsbXMiOnsicm9sZXMiOlsidW1hX3Byb3RlY3Rpb24iXX0sImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoiIn0.3MUdvDFUMumia5AmUJgZatzvds351GqT8am9dCRcG6s",
        "scope": "",
        "session_state": "4c3287b4-f21b-4cb5-a76a-0c255292e0dc",
        "token_type": "bearer"
    },
    "pragma": "no-cache",
    "redirected": false,
    "set_cookie": "KC_RESTART=; Version=1; Expires=Thu, 01-Jan-1970 00:00:10 GMT; Max-Age=0; Path=/auth/realms/sunbird/; HttpOnly",
    "status": 200,
    "url": "https://alex-sb.dev.verlab.io/auth/realms/sunbird/protocol/openid-connect/token",
    "vary": "Accept-Encoding"
}

MSG:

OK (2676 bytes)

TASK [post-install : set_fact] *************************************************
ok: [localhost] => {
    "ansible_facts": {
        "keycloak_access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI4NklnYTA1UUVXRWxOcF9aMzkyQ3BzZDBTX3gzWm1nZkczNFp3cGFza2pRIn0.eyJqdGkiOiJiNmQ0NzI3OS05MDMzLTRkNTAtYWY0YS0yMjlmZDUxZjNjYmMiLCJleHAiOjE2ODYxNTc2NzYsIm5iZiI6MCwiaWF0IjoxNjg2MTE0NDc2LCJpc3MiOiJodHRwczovL2FsZXgtc2IuZGV2LnZlcmxhYi5pby9hdXRoL3JlYWxtcy9zdW5iaXJkIiwiYXVkIjpbInJlYWxtLW1hbmFnZW1lbnQiLCJhY2NvdW50Il0sInN1YiI6IjI5NTAwYmY3LTg2NGYtNDFhYS05MGE4LWM0MjBlOWUwN2JkMCIsInR5cCI6IkJlYXJlciIsImF6cCI6ImxtcyIsImF1dGhfdGltZSI6MCwic2Vzc2lvbl9zdGF0ZSI6IjRjMzI4N2I0LWYyMWItNGNiNS1hNzZhLTBjMjU1MjkyZTBkYyIsImFjciI6IjEiLCJhbGxvd2VkLW9yaWdpbnMiOlsiaHR0cHM6Ly9hbGV4LXNiLmRldi52ZXJsYWIuaW8iXSwicmVhbG1fYWNjZXNzIjp7InJvbGVzIjpbIm9mZmxpbmVfYWNjZXNzIiwidW1hX2F1dGhvcml6YXRpb24iXX0sInJlc291cmNlX2FjY2VzcyI6eyJyZWFsbS1tYW5hZ2VtZW50Ijp7InJvbGVzIjpbIm1hbmFnZS11c2VycyJdfSwibG1zIjp7InJvbGVzIjpbInVtYV9wcm90ZWN0aW9uIl19LCJhY2NvdW50Ijp7InJvbGVzIjpbIm1hbmFnZS1hY2NvdW50IiwibWFuYWdlLWFjY291bnQtbGlua3MiLCJ2aWV3LXByb2ZpbGUiXX19LCJzY29wZSI6IiIsImNsaWVudElkIjoibG1zIiwiY2xpZW50SG9zdCI6IjE3Mi4xNi4yLjE3MiIsInByZWZlcnJlZF91c2VybmFtZSI6InNlcnZpY2UtYWNjb3VudC1sbXMiLCJjbGllbnRBZGRyZXNzIjoiMTcyLjE2LjIuMTcyIiwiZW1haWwiOiJzZXJ2aWNlLWFjY291bnQtbG1zQHBsYWNlaG9sZGVyLm9yZyJ9.UF-T3xE-pUKleE0LCBBcpJwjKO9MWE0nbjCi67JnezVvU4jPvXceKr2wcGySf-2tydaIG12zVwsKGRUCTD_DT1dDbeSriJztxhCBXELRu9AycJ489J6doqrFUQECFrESCQeqPkjQ8Jn1tjHzeqqFjZvn8dATrGrYUd2HZmP5auMFHv8N3cgrlgBkLwgHQYU_Qk5IgpcfViEqGMNZf1utZkPPeoh4RFckpD4q6IssN82XUv4VzbgiHW6dTgJvQfzhhY7kqlPZVvZApFYwalMS-1OLit5oJT5ynTynIXf4oJoTNpiCBVAf9TVnPEl9Xy-pQHnsNJ1IAKWff-Y4fIHacQ"
    },
    "changed": false
}

TASK [post-install : debug] ****************************************************
ok: [localhost] => {
    "keycloak_access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI4NklnYTA1UUVXRWxOcF9aMzkyQ3BzZDBTX3gzWm1nZkczNFp3cGFza2pRIn0.eyJqdGkiOiJiNmQ0NzI3OS05MDMzLTRkNTAtYWY0YS0yMjlmZDUxZjNjYmMiLCJleHAiOjE2ODYxNTc2NzYsIm5iZiI6MCwiaWF0IjoxNjg2MTE0NDc2LCJpc3MiOiJodHRwczovL2FsZXgtc2IuZGV2LnZlcmxhYi5pby9hdXRoL3JlYWxtcy9zdW5iaXJkIiwiYXVkIjpbInJlYWxtLW1hbmFnZW1lbnQiLCJhY2NvdW50Il0sInN1YiI6IjI5NTAwYmY3LTg2NGYtNDFhYS05MGE4LWM0MjBlOWUwN2JkMCIsInR5cCI6IkJlYXJlciIsImF6cCI6ImxtcyIsImF1dGhfdGltZSI6MCwic2Vzc2lvbl9zdGF0ZSI6IjRjMzI4N2I0LWYyMWItNGNiNS1hNzZhLTBjMjU1MjkyZTBkYyIsImFjciI6IjEiLCJhbGxvd2VkLW9yaWdpbnMiOlsiaHR0cHM6Ly9hbGV4LXNiLmRldi52ZXJsYWIuaW8iXSwicmVhbG1fYWNjZXNzIjp7InJvbGVzIjpbIm9mZmxpbmVfYWNjZXNzIiwidW1hX2F1dGhvcml6YXRpb24iXX0sInJlc291cmNlX2FjY2VzcyI6eyJyZWFsbS1tYW5hZ2VtZW50Ijp7InJvbGVzIjpbIm1hbmFnZS11c2VycyJdfSwibG1zIjp7InJvbGVzIjpbInVtYV9wcm90ZWN0aW9uIl19LCJhY2NvdW50Ijp7InJvbGVzIjpbIm1hbmFnZS1hY2NvdW50IiwibWFuYWdlLWFjY291bnQtbGlua3MiLCJ2aWV3LXByb2ZpbGUiXX19LCJzY29wZSI6IiIsImNsaWVudElkIjoibG1zIiwiY2xpZW50SG9zdCI6IjE3Mi4xNi4yLjE3MiIsInByZWZlcnJlZF91c2VybmFtZSI6InNlcnZpY2UtYWNjb3VudC1sbXMiLCJjbGllbnRBZGRyZXNzIjoiMTcyLjE2LjIuMTcyIiwiZW1haWwiOiJzZXJ2aWNlLWFjY291bnQtbG1zQHBsYWNlaG9sZGVyLm9yZyJ9.UF-T3xE-pUKleE0LCBBcpJwjKO9MWE0nbjCi67JnezVvU4jPvXceKr2wcGySf-2tydaIG12zVwsKGRUCTD_DT1dDbeSriJztxhCBXELRu9AycJ489J6doqrFUQECFrESCQeqPkjQ8Jn1tjHzeqqFjZvn8dATrGrYUd2HZmP5auMFHv8N3cgrlgBkLwgHQYU_Qk5IgpcfViEqGMNZf1utZkPPeoh4RFckpD4q6IssN82XUv4VzbgiHW6dTgJvQfzhhY7kqlPZVvZApFYwalMS-1OLit5oJT5ynTynIXf4oJoTNpiCBVAf9TVnPEl9Xy-pQHnsNJ1IAKWff-Y4fIHacQ"
}

TASK [post-install : uri] ******************************************************
fatal: [localhost]: FAILED! => {
    "access_control_allow_origin": "*",
    "changed": false,
    "connection": "close",
    "content": "{\"id\":\"api.org.create\",\"ver\":\"v1\",\"ts\":\"2023-06-07 05:07:57:052+0000\",\"params\":{\"resmsgid\":\"655593f047ab6ead335e87379c2f3c71\",\"msgid\":\"655593f047ab6ead335e87379c2f3c71\",\"err\":\"UOS_ORGCRT0018\",\"status\":\"FAILED\",\"errmsg\":\"Invalid organisationType: board. Valid values are: [].\"},\"responseCode\":\"CLIENT_ERROR\",\"result\":{}}",
    "content_length": "319",
    "content_type": "application/json",
    "date": "Wed, 07 Jun 2023 05:07:57 GMT",
    "elapsed": 0,
    "json": {
        "id": "api.org.create",
        "params": {
            "err": "UOS_ORGCRT0018",
            "errmsg": "Invalid organisationType: board. Valid values are: [].",
            "msgid": "655593f047ab6ead335e87379c2f3c71",
            "resmsgid": "655593f047ab6ead335e87379c2f3c71",
            "status": "FAILED"
        },
        "responseCode": "CLIENT_ERROR",
        "result": {},
        "ts": "2023-06-07 05:07:57:052+0000",
        "ver": "v1"
    },
    "redirected": false,
    "referrer_policy": "origin-when-cross-origin, strict-origin-when-cross-origin",
    "status": 400,
    "url": "https://alex-sb.dev.verlab.io/api/org/v1/create",
    "via": "kong/0.14.1",
    "x_content_type_options": "nosniff",
    "x_frame_options": "DENY",
    "x_kong_proxy_latency": "34",
    "x_kong_upstream_latency": "503",
    "x_permitted_cross_domain_policies": "master-only",
    "x_ratelimit_limit_hour": "5000",
    "x_ratelimit_remaining_hour": "4999",
    "x_xss_protection": "1; mode=block"
}

MSG:

Status code was 400 and not [200]: HTTP Error 400: Bad Request

PLAY RECAP *********************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // stage
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (slack_notify)
[Pipeline] sh
+ git rev-parse --short HEAD
[Pipeline] sh
+ git name-rev --name-only HEAD
+ cut -d / -f1
+ rev
+ rev
[Pipeline] sh
+ echo OpsAdministration/alex/dev/Core/PostInstallScript
[Pipeline] echo
Could not find env specific Slack channel. Check for global slack channel..
[Pipeline] echo
Could not find slack environment variable. Skipping slack notification..
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (email_notify)
[Pipeline] sh
+ echo OpsAdministration/alex/dev/Core/PostInstallScript
[Pipeline] echo
Could not find env specific email group. Check for global email group..
[Pipeline] echo
Could not find global email group variable. Skipping email notification..
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 2
Finished: FAILURE

**Comment by chitranshu-keshav** [comment]
@santhosh-tg @tsprasath can you please get back on this

**Reply by tsprasath** [reply]
@chitranshu-keshav I have connected to @srikanth-6520 and resolved the issue.

**Reply by srikanth-6520** [reply, marked as answer]
@santhosh-tg issue got resolve , thank you.
